test/functional/legacy-fixtures/native-dialogs-handling/pages/cross-domain.html

```diff
@@ -4,6 +4,6 @@
     <title>Cross-domain iframe page</title>
 </head>
 <body>
-<iframe id="iframe" src=http://localhost:3001/legacy-fixtures/native-dialogs-handling/pages/index.html></iframe>
+<iframe id="iframe" src=http://localhost:3001/legacy-fixtures/api/es-next/native-dialogs-handling/pages/index.html></iframe>
```

Comment:
This is a legacy-api functional tests file. I suppose it was changes by a mistake